go/api/base_client.go

```diff
@@ -1348,3 +1348,11 @@ func (client *baseClient) ZPopMaxWithCount(key string, count int64) (map[Result[
 	}
 	return handleStringDoubleMapResponse(result)
 }
+
+func (client *baseClient) Echo(message string) (Result[string], error) {
+	result, err := client.executeCommand(C.Echo, []string{message})
+	if err != nil {
+		return CreateNilStringResult(), err
+	}
+	return handleStringOrNullResponse(result)
```

Comment:
```suggestion
	return handleStringResponse(result)
```

Reply:
fix in commit 2ac4523fe5714d35e32304c4d6f7cf4e9ae74067